libs/design/src/atoms/form/checkbox/checkbox.component.html

```diff
@@ -0,0 +1,10 @@
+<input type=checkbox
```

Comment:
`type="checkbox"`

Reply:
Okay fixed in a19666774303a5986f9a4617acdc5a58ec028acf